Consul/Lock.cs

```diff
@@ -251,23 +252,24 @@ public async Task<CancellationToken> Acquire(CancellationToken ct)
 
                     var qOpts = new QueryOptions()
                     {
-                        WaitTime = Opts.LockWaitTime
+                        WaitTime = Opts.LockWaitTime,
                     };
 
                     var attempts = 0;
-                    var start = DateTime.UtcNow;
```

Comment:
`DateTime` arithmetic is not a good way of calculating elapsed time, `StopWatch` is more accurate.